desc/compute/_qs.py

```diff
@@ -2,131 +2,296 @@
+    GI = data["G"] + data["iota"] * data["I"]
+    w = transforms["w"].transform(data["w_mn"])
+    data["nu"] = (w - data["I"] * data["lambda"]) / GI
     return data
 
 
-def compute_quasisymmetry_error(params, transforms, profiles, data=None, **kwargs):
-    """Compute quasi-symmetry triple product and two-term errors.
+@register_compute_fun(
+    name="nu_t",
+    label="\\partial_{\\theta} \\nu",
+    units="rad",
+    units_long="radians",
+    description="Boozer toroidal stream function, derivative wrt poloidal angle",
+    dim=1,
+    params=[],
+    transforms={"w": [[0, 1, 0]]},
+    profiles=[],
+    function_of="rtz",
+    data=["w_mn", "G", "I", "iota", "lambda_t"],
+)
+def _nu_t(params, transforms, profiles, data, **kwargs):
+    GI = data["G"] + data["iota"] * data["I"]
+    w_t = transforms["w"].transform(data["w_mn"], dr=0, dt=1, dz=0)
+    data["nu_t"] = (w_t - data["I"] * data["lambda_t"]) / GI
+    return data
+
 
-    f_C computation assumes transform grids are a single flux surface.
-    """
-    data = compute_B_dot_gradB(
-        params,
-        transforms,
-        profiles,
-        data=data,
-        **kwargs,
-    )
-    data = compute_boozer_magnetic_field(
-        params,
-        transforms,
-        profiles,
-        data=data,
-        **kwargs,
+@register_compute_fun(
+    name="nu_z",
+    label="\\partial_{\\zeta} \\nu",
+    units="rad",
+    units_long="radians",
+    description="Boozer toroidal stream function, derivative wrt toroidal angle",
+    dim=1,
+    params=[],
+    transforms={"w": [[0, 0, 1]]},
+    profiles=[],
+    function_of="rtz",
+    data=["w_mn", "G", "I", "iota", "lambda_z"],
+)
+def _nu_z(params, transforms, profiles, data, **kwargs):
+    GI = data["G"] + data["iota"] * data["I"]
+    w_z = transforms["w"].transform(data["w_mn"], dr=0, dt=0, dz=1)
+    data["nu_z"] = (w_z - data["I"] * data["lambda_z"]) / GI
+    return data
+
+
+@register_compute_fun(
+    name="theta_B",
+    label="\\theta_{B}",
+    units="rad",
+    units_long="radians",
+    description="Boozer poloidal angular coordinate",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    function_of="rtz",
+    data=["theta", "lambda", "iota", "nu"],
+)
+def _theta_B(params, transforms, profiles, data, **kwargs):
+    data["theta_B"] = data["theta"] + data["lambda"] + data["iota"] * data["nu"]
+    return data
+
+
+@register_compute_fun(
+    name="zeta_B",
+    label="\\zeta_{B}",
+    units="rad",
+    units_long="radians",
+    description="Boozer toroidal angular coordinate",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    function_of="rtz",
+    data=["zeta", "nu"],
+)
+def _zeta_B(params, transforms, profiles, data, **kwargs):
+    data["zeta_B"] = data["zeta"] + data["nu"]
+    return data
+
+
+@register_compute_fun(
+    name="sqrt(g)_B",
+    label="\\sqrt{g}_{B}",
+    units="~",
+    units_long="None",
+    description="Jacobian determinant of Boozer coordinates",
```

Comment:
Similar comment to a previous one about the Jacobians. I suggest the description: 
`"Jacobian of the Boozer coordinate system transformation"`